js/blocks.js

```diff
@@ -519,7 +518,8 @@ function Blocks () {
     };
 
     /*
-     * Adjust the clamp of expandable block
+     *  Adjust the size of the clamp in an expandable block when block are inserted into (or removed from) the child flow. This is a
```

Comment:
Can you please spread this comment across more lines so we don't need to scroll to read it?